Frediflexta/StackOverflowLite
# Fix test coverage report to travis

### What does this PR do?
> Design a mock db for testing out the APIs' and also add test coverage to travis
#### Description of Task to be completed?
> Use an array of objects to store questions and answers with dummy data, and integrate tests with travis and get coverage as well
#### How should this be manually tested?
> Visit https://travis-ci.org/Frediflexta/StackOverflowLite and see test coverage and report
#### What are the relevant pivotal tracker stories?
> #159792716
